executor/bitmap.h

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+#include <fcntl.h>
+#include <stdio.h>
+
+#if GOOS_linux || GOOS_freebsd || GOOS_netbsd || GOOS_openbsd || GOOS_akaros
+#include <sys/mman.h>
+#endif
+
+static byte* bitmap;
+static uint32 pcstart;
+static uint32 pcsize;
+static uint32 pcend;
+
+inline void init_bitmap()
+{
+#if SYZ_EXECUTOR_USES_SHMEM
+	int f = open("/syz-cover-bitmap", O_RDONLY);
```

Comment:
Need to close the fd.
Otherwise it leaks a file descriptor and will break this test:
https://github.com/google/syzkaller/blob/master/sys/linux/test/fd_assignment
and change behavior of some existing programs in corpus.